rocketpool-cli/service/config/standard-layout.go

```diff
@@ -205,3 +205,23 @@ func (layout *standardLayout) mapParameterizedFormItems(params ...*parameterized
 		layout.parameters[param.item] = param
 	}
 }
+
+// Sets up a handler to return to the specified homePage when the user presses escape on the layout.
+func (layout *standardLayout) setupEscapeReturnHomeHandler(md *mainDisplay, homePage *page) {
```

Comment:
FYI: I found 10 copies of this code throughout the codebase and replaced them all with this function.